fastparse/shared/src/main/scala/fastparse/Api.scala

```diff
@@ -107,8 +107,12 @@ class ByteApi() extends Api[Byte, Array[Byte]]() {
     val bytes = all.P(byteSep ~ byte.rep(sep = byteSep)).map(_.toArray)
   }
 
-  def strToBytes(s: String): Array[Byte] = {
-    BytesParser.bytes.parse(s).get.value
+  /**
+    * Helper method to convert a space-separated string of hexidecimal bytes
+    * (e.g. "01 bc 21 04") into the corresponding Array[Byte]
```

Comment:
Maybe we can make case sensitivity, but the "0x" prefix feature looks okay for me.